### PR TITLE
Add EU exit business finder metadata tags to all relevant schemas

### PR DESCRIPTION
**Pull request by barrucadu**
Should match Rummager schema stuff:

- https://github.com/alphagov/rummager/blob/master/config/schema/field_definitions.json
- https://github.com/alphagov/rummager/search?q=appear_in_find_eu_exit_guidance_business_finder&unscoped_q=appear_in_find_eu_exit_guidance_business_finder

[Trello card](https://trello.com/c/Z3jG94IX/68-add-tags-to-the-content-items)

### CODE REVIEW

File: formats/shared/definitions/eu_exit_business_finder_metadata.jsonnet [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
```
Comment (thomasleese) [inline formats/shared/definitions/eu_exit_business_finder_metadata.jsonnet]: I don't think we need this file anymore.

**Comment by thomasleese** [comment]
I guess we don't need this PR now?

**Comment by rubenarakelyan** [comment]
It would be good to split your changes and the ones made by running `rake` into separate commits, otherwise it's very difficult to see what you changed vs. what the script changed.

**Comment by barrucadu** [comment]
I don't think we need this PR for now, we'll need to think about how stuff is actually getting tagged.

**Pull request closed without merging**